src/include/gurt/telemetry_common.h

```diff
@@ -10,13 +10,122 @@
 
 #define D_TM_VERSION			1
 #define D_TM_MAX_NAME_LEN		256
-#define D_TM_MAX_SHORT_LEN		64
-#define D_TM_MAX_LONG_LEN		1024
+#define D_TM_MAX_DESC_LEN		64
+#define D_TM_MAX_UNIT_LEN		16
 #define D_TM_TIME_BUFF_LEN		26
 
 #define D_TM_SHARED_MEMORY_KEY		0x10242048
 #define D_TM_SHARED_MEMORY_SIZE		(1024 * 1024)
 
+/**
+ * The following definitions are suggested strings for units that may be used
+ * when explicitly calling d_tm_add_metric() to initialize a metric before use.
+ */
```

Comment:
Looking at these constants, I feel like they're unnecessary, seeing that units is an arbitrary string. I wouldn't block on it, but unless we're planning on keying off the units for something, I'd suggest pulling these out.

Reply:
The intention was to provide string constants that gave some standardized options.  Having the standardized options might help avoid creativity when specifying the units.